lib/spack/docs/extensions.rst

```diff
@@ -62,7 +61,7 @@ paths to ``config.yaml``. In the case of our example this means ensuring that:
 
    config:
      extensions:
-     - /home/user/tmp/spack-scripting
+     - ~/tmp/spack-scripting
```

Comment:
Last I recall, `~` doesn't actually work in config files, which is why things were switched to `/home/user`. If that's no longer the case, that's great!